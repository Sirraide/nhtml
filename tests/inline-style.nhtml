div%[display: flex; flex-direction: row] {
    %[display: flex; flex-direction: row] (test)
}
