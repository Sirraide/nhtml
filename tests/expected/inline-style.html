<div style='display: flex; flex-direction: row'>
    <div style='display: flex; flex-direction: row'>test</div>
</div>
